mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/cli/OrientDBBootstrapInitializerImpl.java

```diff
@@ -273,7 +226,7 @@ protected void initCluster(MeshOptions options, PostProcessFlags flags, boolean
 		boolean active = false;
 		while (!active) {
 			log.info("Waiting for hazelcast to become active");
-			active = manager.getHazelcastInstance().getLifecycleService().isRunning();
+			active = ((HazelcastClusterManager) getClusterManager()).getHazelcastInstance().getLifecycleService().isRunning();
```

Comment:
We should avoid this cast (I saw the same cast in the other bootstrap initializer implementation).
Maybe we should move method HazelcastInstance getHazelcast(); from interface OrientDBClusterManager to ClusterManager and use that here and in the other bootstrap initializer implementation to get the Hazelcast instance?

Reply:
I went with the generic ClusterManager implementation because we might use different implementations for different versions (e.g. ZookeperClusterManager), and I wanted to avoid adding the vertx-hazelcast dependency in the mdm-api.
But if that's not a problem, we can add a dependency to vertx-hazelcast and return the hazelcast implementation, or add a method returning the hazelcast instance as suggested. Both are fine from my point of view

Reply:
Currently, there are no plans to use a different cluster manage, and if we do that sometime in the future, we need to change a lot more than just that, so I would go for the simplest and cleanest solution for now.

Reply:
I went with exposing the HazelcastInstance in ClusterManager